src/XRP/issued-currency-client.ts

```diff
@@ -349,25 +351,108 @@ export default class IssuedCurrencyClient {
     amount: string,
     wallet: Wallet,
   ): Promise<TransactionResult> {
-    if (!XrpUtils.isValidXAddress(issuerXAddress)) {
+    return await this.sendTrustSetTransaction(
+      issuerXAddress,
+      currencyName,
+      amount,
+      undefined,
+      wallet,
+    )
+  }
+
+  /**
+   * Creates an authorized trust line between this XRPL account and another account.
+   * An authorized trust line will only be created if `requiredAuthorizedTrustlines` is called first.
```

Comment:
I refreshed my reading on authorized trust lines and I propose the documentation for this method reads more like this:
``` Creates an authorized trust line between this XRPL account (issuing account) and another account.  Note that the other account must also create a trust line to this issuing account in order to establish a trust line with a non-zero limit.  If this method is called before the other account creates a trust line, a trust line with a limit of 0 is created.  However, this is only true if this issuing account has already required Authorized Trustlines (see https://xrpl.org/authorized-trust-lines.html), otherwise no trust line is created. ```